docs/content/dagster-cloud/getting-started.mdx

```diff
@@ -1,35 +1,341 @@
+
+**Using the GitHub app**
+
+1. Click the **Import an existing project** tab.
+2. Sign in to your GitHub account, if prompted.
+3. In Dagster Cloud, define the following:
+   - **Git scope** - Select the organization or account to create the new repository in.
+   - **Repository Name** - Enter a name for the new repository.
+   - Check the **Make git repository private** box to make the repository private.
+4. Click **Deploy**.
+
+When finished, [continue to the next step](#step-4-set-up-cicd).
+
+**Using the dagster-cloud CLI**
+
+<Note>
+  Using the dagster-cloud CLI requires a recent version of Python 3 and Docker.
+</Note>
+
+To complete this step using the CLI, you can use your own Dagster code or the [Dagster Cloud Serverless Quickstart](https://github.com/dagster-io/dagster-cloud-serverless-quickstart). The quickstart is a template with everything you need to get started using Serverless in Dagster Cloud.
+
+1. If using the template, click [**Use this template** in the repository](https://github.com/dagster-io/dagster-cloud-serverless-quickstart). Finish setting up the repository on GitHub and clone it to your local environment.
+
+2. To install the dagster-cloud CLI, run:
+
+   ```shell
+   pip install dagster-cloud
+   ```
+
+3. Complete the CLI setup:
+
+   1. Complete the authentication step, including [creating a user token](/dagster-cloud/account/managing-user-agent-tokens#managing-user-tokens).
+
+   2. **If using your own Dagster code**, make sure to add `dagster-cloud` as a dependency in `setup.py`. [Click here for an example](https://github.com/dagster-io/dagster-cloud/blob/main/dagster-cloud-examples/setup.py). This is already done for you if using the Serverless quickstart template.
+
+4. Run the following to deploy your code to Dagster Cloud:
+
+   ```shell
+   dagster-cloud serverless deploy \
+         --location-name "<LOCATION_NAME>" \
+         --package-name <PACKAGE_NAME> \
+   ```
+
+When finished, [continue to the next step](#step-4-set-up-cicd).
+
+</TabItem>
+</TabGroup>
+</TabItem>
+<TabItem name="Hybrid">
+
+### Hybrid
+
+To set up Hybrid deployment and deploy your code, you'll need to:
+
+1. Set up an agent
+2. Add the code location in Dagster Cloud
+
+**Step 3.1: Create a Hybrid agent**
+
+1. The modal that displays will contain a pre-generated [agent token](/dagster-cloud/account/managing-user-agent-tokens) and info about the agents currently supported by Dagster Cloud.
+2. Follow the steps for setting up and deploying your agent:
+
+   - [Amazon Elastic Container Service (ECS)](/dagster-cloud/deployment/agents/amazon-ecs/creating-ecs-agent-new-vpc)
+   - [Docker](/dagster-cloud/deployment/agents/docker)
+   - [Kubernetes](/dagster-cloud/deployment/agents/kubernetes/configuring-running-kubernetes-agent)
+   - [Local](/dagster-cloud/deployment/agents/local)
+
+Once Dagster Cloud receives an agent heartbeat, click **Continue** to move on to the next step.
+
+**Step 3.2: Add a Dagster Cloud code location**
```

Comment:
This is the step that's currently in the NUX, so I opted to keep it. Step 4 goes into other options for code location deployment and setting up CD.